### DISCUSSION TITLE
Problema riconoscimento Shelly in Home Assistant

### DISCUSSION THREAD

**Discussion by umd973** [indomus/forum#2219] [HUB: Home Assistant]
Ciao a tutti, sto muovendo i primi passi con Home Assistant e ho provato ad installarlo su un PC Windows 10 per fare un po di pratica. Ho seguito la guida del sito ed installato HA Core con Docker. Il tutto sembra funzionare correttamente ma non riesco ad integrare i componenti Shelly che ho nella mia rete. Ho installato, configurato e regolarmente gestito tramite app shelly un EM e diversi Shelly 1, ma HA non me li rileva nè col componente nativo integrato nè con lo Shelly for Hass. Per completare il quadro vi dico che ho provato a rifare l'installazione su un PC con Windows 11 e qui tutto funziona regolarmente nel senso che gli Shelly sono visti da HA e sono riuscito ad integrarli. Qualcuno saprebbe aiutarmi per capire che errore sto commettendo sulla macchina con Windows 10?
Grazie mille!

**Comment by snakuzzo** [comment]
Con Docker su Windows non funzionerà mai l'autodiscovery, perché network mode host puoi impostarlo solo su Linux. Leggi [QUI](https://docs.docker.com/network/#network-drivers).

Devi inserirli a mano con l'IP:

Vai su Settings > Device and Services > Add Integration > Shelly > Inserisci l'IP di ogni singolo Shelly

**Reply by umd973** [reply]
Ciao si quando lancio l'integrazione nativa di shelly per HA mi chiede proprio l'ip. Inserisco manualmente l'ip (statico) dello shelly ma mi dice che è impossibile connettersi. Però lo shelly è correttamente sulla rete e lo raggiungo tranquillamente dal browser.

**Reply by snakuzzo** [reply]
mi spiace, ma non ho Windows per provare.
Comunque leggi [QUI](https://community.home-assistant.io/t/shelly-integration-fails-to-connect-to-shelly-2-5-after-windows-update-kb5015827/440323)
